core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -52,6 +52,11 @@ THE SOFTWARE.
       If given, this page is only made available to users that has the specified permission.
       (The permission will be checked against the "it" object.)
     </st:attribute>
+    <st:attribute name="sidepanel" use="optional">
```

Comment:
I think there might be a `default` attribute possible on this? Might make other logic easier if you could default it to `true`.